server/routes/serviceProviderReferrals/editSessionForm.ts

```diff
@@ -26,6 +45,8 @@ export default class EditSessionForm {
       paramsForUpdate: {
         appointmentTime: dateResult.value.toISOString(),
         durationInMinutes: durationResult.value.minutes!,
+        appointmentDeliveryType: appointmentDeliveryType.value!,
```

Comment:
does the validator handle the case where this is null of undefined?

Reply:
It's impossible for it to be undefined because the FormValidation will cause an error to be created if it is null. This is following on the conventions for `durationInMinutes` and `appointmentTime` which also use `!` despite it being impossible.

In my opinion the `!` is not necessary because the underlying value is not defined as `undefinable`but is an additional assertation that if it's undefined then something really bad has gone on. Would you like me to remove it?